validator-rust/src/components/mechanism_gaussian.rs

```diff
@@ -30,11 +30,54 @@ impl Component for proto::GaussianMechanism {
             .ok_or_else(|| Error::from("aggregator: missing"))?;
 
         // sensitivity must be computable
-        aggregator.component.compute_sensitivity(
+        let sensitivity_values = aggregator.component.compute_sensitivity(
             &privacy_definition,
             &aggregator.properties,
             &SensitivitySpace::KNorm(2))?;
 
+        let sensitivities = sensitivity_values.array()?.f64()?;
+        let usages = broadcast_privacy_usage(&self.privacy_usage, sensitivities.len())?;
+        let epsilons = usages.iter().map(get_epsilon).collect::<Result<Vec<f64>>>()?;
+        let deltas = usages.iter().map(get_delta).collect::<Result<Vec<f64>>>()?;
+
+        // epsilons must be greater than 0 and less than 1.
+        for epsilon in epsilons.into_iter(){
+            if epsilon <= 0.0 || epsilon >= 1.0{
+                return Err("epsilon: privacy parameter epsilon must be greater than 0".into());
+            };
+            if epsilon >= 1.0   {
+                println!("Warning: A privacy parameter of epsilon = {} is in use. Privacy is only \
+                guaranteed for the Gaussian mechanism as implemented in the rust runtime for epsilon \
+                between 0 and 1.", epsilon);
+            }
+        }
+
+        // Check delta value; checks depend on whether or not number of records is statically known.
+        match data_property.num_records {
+            Some(_n) => {
```

Comment:
When you match, you are declaring a new variable named "_n" here with the value inside the num_records option. Instead of reassigning "_n" to "n", you can just do `Some(n)`. Typically you prefix a variable with `_n` if you are ignoring the value.